### ISSUE TITLE
Static final fields are transformed to method calls

### ISSUE THREAD

**Issue by bjansen** [ceylon/ceylon.tool.converter.java2ceylon#16]
This

``` java
    public static final int VARCHAR_SIZE = 255;
    public static final int TEXT_SIZE = 8192;
    public static final String DATE_FORMAT = "yyyy-MM-dd HH:mm:ss.SSS";
```

is incorrectly transformed to this:

``` ceylon
    shared Integer \iVARCHAR_SIZE();
    shared Integer \iTEXT_SIZE();
    shared String \iDATE_FORMAT();
```

Expected:

``` ceylon
    shared Integer \iVARCHAR_SIZE = 255;
    shared Integer \iTEXT_SIZE = 8192;
    shared String \iDATE_FORMAT = "yyyy-MM-dd HH:mm:ss.SSS";
```

**Comment by rohitmohan96** [comment]
I get the expected output when I run the converter from command line but get the same problem when I do it from eclipse.

**Comment by rohitmohan96** [comment]
Looks like it gets converted properly only when inside a class.

**Comment by bjansen** [comment]
Oh sorry I know what happened, I didn't copy/paste the whole classe, but just bits of it. Since fields are not supposed to live outside of classes in Java, the parser might have parsed those single fields incorrectly, transforming them to methods.

**Comment by bjansen** [comment]
This means that if we want to fix this issue, we'll have to modify the grammar. We'll also have the same problem if we try to convert a single `for` loop, for example. I'm wondering if we should handle such cases too...

**Comment by rohitmohan96** [comment]
Fixed in 883ef8061b31f87749572805741994e730ca0f4a
